src/EventId.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry;
+
+/**
+ * This class represents an event ID.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+final class EventId
```

Comment:
Would possibly be nice to also add a `getId` getter to this class so you don't have to use a cast or call `__toString()`.

Reply:
It's possible of course, but I don't see a real value in having it because of a few reasons:

- what are the advantages of calling `getId` which returns a `string` vs casting to `string` the object?
- the value object represents the ID, so it doesn't really makes sense imho to have a method that basically returns itself